scarb/src/ops/workspace.rs

```diff
@@ -114,6 +113,15 @@ fn read_workspace_root<'c>(
             .map(AsRef::as_ref)
             .map(|package_path| {
                 let package_manifest = TomlManifest::read_from_path(package_path)?;
+                if package_path != manifest_path
+                    && let Some(member_ws) = package_manifest.get_workspace()
+                    && member_ws.require_audits.is_some()
+                {
+                    return Err(anyhow!(
+                        "only the workspace root may set `[workspace].require-audits`"
```

Comment:
I don't understand this check. 

Scarb run is always contained to a single workspace. So we should never read keys from other workspaces. 

We may (not sure if we should) decide to disallow workspaces intertwining like this altogether - but disallowing using some keys in one of them is pointless. 

Effectively, all you achieve here, is to disallow using the new setting in some workspaces based on values set in other workspaces, which should never happen.